### PR TITLE
feat(Pivot) make rows header in pivot clickable

### PR DESCRIPTION
**Pull request by denaldhushi**

### CODE REVIEW

File: include/utils/utils.php [hunk 1]
```diff
@@ -3904,4 +3904,20 @@ function validateClickHouseSecret($signedvalue, $signedkey, $input) {
 	$computedSignature = base64_encode(hash_hmac('sha256', $input, $signedvalue, true));
 	return ($receivedSignature === $computedSignature);
 }
+
+function getFieldForAdvancedSearch($field, $module) {
```
Comment (joebordes) [inline include/utils/utils.php:3908]: we have this function in Custom View: https://github.com/tsolucio/corebos/blob/master/modules/CustomView/CustomView.php#L530

please use that one

File: Smarty/templates/PivotView.tpl [hunk 2]
```diff
@@ -121,6 +122,24 @@ $(function() {
...
+					let vColRowLabels = vTotalAxisLabels - vTotalRowLabels ;
+					let thHeaderName = $('.pvtAxisLabel').eq($(this).index()+vColRowLabels);
+					const advSearch = [{
+						'columnname': rowFields[thHeaderName.html()],
+						'comparator': 'e',
+						'value': e.target.innerText,
+						'groupid': 1,
+						'columncondition': 'and'
+					}];
+					const urlstring = `${JSON.stringify(advSearch)}&advft_criteria_groups=[null,{"groupcondition":""}]`;
```
Comment (joebordes) [inline Smarty/templates/PivotView.tpl:140]: don't we have to use encodeuricomponent here?  For example, if `e.target.innerText` or the header has a value of `me&you` I think that ampersand will break the URL.

Comment (denaldhushi) [reply Smarty/templates/PivotView.tpl:140]: Updated :D

**Comment by joebordes** [comment]
LGTM. Thanks!

**Pull request merged**